client/signup/flow-actions.ts

```diff
@@ -4,5 +4,6 @@ export const onEnterOnboarding = ( flowName: string ) => {
 	// just to be extra safe since `loadExperimentAssignment` doesn't have the same eligibility check like `useExperiment` does
 	if ( flowName === 'onboarding' ) {
 		loadExperimentAssignment( 'calypso_signup_onboarding_deemphasize_free_plan' );
+		loadExperimentAssignment( 'calypso_signup_onboarding_emphasize_14d_refund' );
```

Comment:
When does the assignment occur? Is this after the user completes account creation and lands on `/start/domains`?

Reply:
Thanks @southp! Ideally, users are assigned after being given a logged-in userid, ie _after_ `/start/user`. This helps ensure variation consistency.

It's not a big deal, since we have an exposure event on the experiment. But best practice is to assign logged-in when possible!

Reply:
Thanks for clarification. In that case, we should remove this line to be safe. I'll also make a note of avoiding to preload experiments here without confirming with the ExPlat team first in the future. 🙇🏼

Reply:
Sounds good! After removal, it may be worth testing out the variations again.

Reply:
Sounds good. Thanks for the insights. I will remove then and confirm.